### PR TITLE
raftstore: use static lease

### PR DESCRIPTION
**Pull request by BusyJay**
So when the cluster can be upgraded rollingly as long as new election timeout is larger than old lease.

@siddontang @hhkbp2 PTAL

### CODE REVIEW

File: src/raftstore/store/config.rs [hunk 1]
```diff
@@ -52,6 +54,8 @@ const DEFAULT_CONSISTENCY_CHECK_INTERVAL: u64 = 0;
 
 const DEFAULT_REPORT_REGION_FLOW_INTERVAL: u64 = 30000; // 30 seconds
 
+const DEFAULT_RAFT_STORE_LEASE: i64 = 5; // 5 seconds
```
Comment (ngaut) [inline src/raftstore/store/config.rs]: s/DEFAULT_RAFT_STORE_LEASE/RAFT_STORE_LEASE_SEC/

File: src/raftstore/store/config.rs [hunk 2]
```diff
@@ -117,6 +121,8 @@ pub struct Config {
     pub consistency_check_tick_interval: u64,
 
     pub report_region_flow_interval: u64,
+    // The lease provided by a successfully proposed and applied entry.
+    pub raft_store_lease: TimeDuration,
```
Comment (siddontang) [inline src/raftstore/store/config.rs]: raft_store_leader_lease

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @hhkbp2

**Comment by hhkbp2** [comment]
Please update(or revise if you like) these comments as well 

https://github.com/pingcap/tikv/blob/master/src/raftstore/store/peer.rs#L609

**Comment by hhkbp2** [comment]
Rest LGTM

File: src/raftstore/store/config.rs [hunk 3]
```diff
@@ -52,6 +54,8 @@ const DEFAULT_CONSISTENCY_CHECK_INTERVAL: u64 = 0;
 
 const DEFAULT_REPORT_REGION_FLOW_INTERVAL: u64 = 30000; // 30 seconds
 
+const DEFAULT_RAFT_STORE_LEASE_SEC: i64 = 5; // 5 seconds
```
Comment (ngaut) [inline src/raftstore/store/config.rs]: 5 is too tiny, how about increase it to 9 ?

**Comment by ngaut** [comment]
Ping

**Pull request merged**